node/sc/bridge_manager_test.go

```diff
@@ -76,6 +76,44 @@ func CheckReceipt(b bind.DeployBackend, tx *types.Transaction, duration time.Dur
 	assert.Equal(t, expectedStatus, receipt.Status)
 }
 
+func handleValueTransfer(t *testing.T, ev IRequestValueTransferEvent, bridgeInfo *BridgeInfo, wg *sync.WaitGroup, backend *backends.SimulatedBackend) {
+	var (
+		tokenType      = ev.GetTokenType()
+		valueOrTokenId = ev.GetValueOrTokenId()
+		from           = ev.GetFrom()
+		to             = ev.GetTo()
+		contractAddr   = ev.GetRaw().Address
+		tokenAddr      = ev.GetTokenAddress()
+		requestNonce   = ev.GetRequestNonce()
+		txHash         = ev.GetRaw().TxHash
+	)
+	fmt.Println("Request Event",
```

Comment:
Can you use t.Log instead of fmt.Println? CI cannot control this out

Reply:
Before changing those outputs, could you elaborate on the relationship between the test output and CI? It just prints the info to ensure the correct values are filled in, not related to the expected output of the test. This question is to understand the need for change.

If the standard output matter to CI, we can simply remove all the print logs.

Reply:
Test output and CI doesn't different, but our CI show limited lines of logs during the test on the initial web page. If we print normal outputs too much, it is annoying when we find the real error logs.

Reply:
So, it is not urgent to remove all stdout during the test. But, we better reduce them when we make a new test case.

Reply:
Okay. Replaced from `fmt.Println` to `t.Log` and `fmt.Printf` to `t.Logf`